### DISCUSSION TITLE
Any way to use this plugin with inline `background-image: url(' ');`

### DISCUSSION THREAD

**Discussion by michaelmannucci** [spatie/statamic-responsive-images#168] [General]
Title is self-explanatory. Can I use this plugin with CSS's `background-image: url('');`? More specifically, with using tailwind: `bg-[url('')]`

**Comment by ncla** [comment]
I think this addon is too overkill if you just want a single image generated. What is the use case here? Why not just use regular Asset fieldtype and plain old Glide tag?

**Comment by DamSha** [comment]
Hey!

I had the same issue, because sometimes I need this image as a background image and sometimes as a responsive image.
For the background-image the solution is simple :
`<section style="background-image: url({{ glide :src="main_image:src:url" }});">` 
- **main_image** is my responsive image field

You can check all available variables with {{ **main_image** | dump }}
